src/AuthenticatedAPIClient/authInterface.js

```diff
@@ -2,6 +2,28 @@ import Cookies from 'universal-cookie';
 import jwtDecode from 'jwt-decode';
 
 
+function getAuthenticatedUserFromDecodedAccessToken(decodedAccessToken) {
+  if (decodedAccessToken === null) {
+    throw new Error('Decoded access token is required to get authenticated user.');
+  }
+
+  const authenticatedUser = {
```

Comment:
```
return {
  userId: decodedAccessToken.user_id,
  username: decodedAccessToken.preferred_username,
  roles: decodedAccessToken.roles ? decodedAccessToken.roles : [],
  administrator: decodedAccessToken.administrator,
};
```
Maybe? :)

Reply:
I can update.  I refactored it to this when debugging.